### ISSUE TITLE
Add benchmarks for workloads running in the guest

### ISSUE THREAD

**Issue by andreeaflorescu** [firecracker-microvm/firecracker#1074]
We need to understand if we get any performance impact when running on AMD vs Intel. While it is hard to come up with meaningful workloads for all scenarios, we should still have some basic workloads in our CI to understand where we're standing.

**Comment by andreeaflorescu** [comment]
The priority is medium here because it has to be done before the AMD milestone is declared completed. It isn't blocking anything other than that.

**Comment by AlexandruCihodaru** [comment]
Superseded by work done for performance testing done in 2021 and planned for 2022.
